tools/pythonpkg/tests/fast/pandas/test_pandas_arrow.py

```diff
@@ -56,9 +56,9 @@ def test_mixed_columns(self):
         python_df = pd.DataFrame({'a': pd.Series(['test', [5, 4, 3], {'a': 42}])}).convert_dtypes()
 
         df = pd.concat([numpy_df['a'], arrow_df['a'], python_df['a']], axis=1, keys=['numpy', 'arrow', 'python'])
-        assert isinstance(df.dtypes[0], pd.core.arrays.integer.IntegerDtype)
-        assert isinstance(df.dtypes[1], pd.core.arrays.arrow.dtype.ArrowDtype)
-        assert isinstance(df.dtypes[2], np.dtype('O').__class__)
+        assert isinstance(df.dtypes['numpy'], pd.Int64Dtype)
```

Comment:
I'm not entirely sure about this, I have had issues in the past where on Linux and Mac, Numpy does not agree on long/int so ideally I would prefer to be able to do `pd.IntegerDtype` but that import doesn't work

Reply:
Yea it caused an issue, using `is_integer_dtype` now